ui/scripts/ui/widgets/listView.js

```diff
@@ -1922,7 +1922,14 @@
 
         // List view header actions
         if (listViewData.actions) {
+            // If a preFilter is set, then get the values. Else set this to null
+            var filteredActions = listViewData.actionPreFilter == undefined ?
+                        null : listViewData.actionPreFilter();
```

Comment:
Why not set to an empty list rather than ``null`` to avoid a potential null error?